app/services/workflow_errors_reporter.rb

```diff
@@ -0,0 +1,31 @@
+require 'faraday'
+# This class will send a put ReST call to the preservationWF via faraday. This also parses
+# out a shorter error message, that only includes the moab validation error.
+class WorkflowErrorsReporter
+
+  def self.update_workflow(druid, error_message)
+    error_message.select { |error| request(druid, 'moab-valid', moab_error(error)) if error.key?(PreservedObjectHandlerResults::INVALID_MOAB) }
+  end
+
+  private_class_method def self.conn
+    if Settings.workflow_services.url.present?
+      @connection ||= Faraday.new(url: Settings.workflow_services.url)
+      Rails.logger.log(warn, 'no workflow hookup - assume you are in test or dev environment')
+    end
+    @connection
+  end
+
+  private_class_method def self.request(druid, process_name, error_message)
```

Comment:
"request" is for the faraday http request, yes?  Would "send_request" be clearer or less clear as a name?  

"process_name" ain't bad, and I know in the context of workflows they use the term "process".  I wonder if we think "workflow_step" or "wf_step" is clearer or less clear.

and I think we may have to do this here:  (can't figure out how to only assign it once and also print warnings -- see previous comment)

```ruby
 private_class_method def self.request(druid, process_name, error_message)
   if conn.is_a?(String)
      logger.warn("{#druid} would write #{error_message} for wf step #{process_name} but #{conn}")
      return
  else
    ...
   conn.put do |request|
  ...
  end
end
```